lib/http-bridge/test/integration/Cardano/Wallet/HttpBridge/NetworkSpec.hs

```diff
@@ -55,9 +57,12 @@ spec = do
     describe "Happy paths" $ around withBridge $ do
         it "get unstable blocks for the unstable epoch" $ \(bridge, _) -> do
             let action = runExceptT $ do
-                    (SlotId ep sl) <- slotId <$> networkTip' bridge
+                    (SlotId (EpochNo ep) (SlotNo sl))
+                        <- slotId <$> networkTip' bridge
                     let sl' = if sl > 2 then sl - 2 else 0
```

Comment:
minor note but, the unwrapping was unnecessary here I think. Since we have a `Num` instance on both `EpochNo` and `SlotNo`, we could have written that without a change I think.